engine/src/main/java/com/redhat/cloud/notifications/exports/transformers/event/CSVEventTransformer.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.cloud.notifications.exports.transformers.event;
+
+import com.redhat.cloud.notifications.exports.ResultsTransformer;
+import com.redhat.cloud.notifications.models.Event;
+
+import java.time.ZoneOffset;
+import java.util.List;
+
+public final class CSVEventTransformer implements ResultsTransformer {
+    /**
+     * Transforms the given list of events to CSV.
+     * @param events the list of events to transform.
+     * @return a {@link String} with the transformed contents.
+     */
+    @Override
+    public String transformEvents(final List<Event> events) {
+        final StringBuilder builder = new StringBuilder();
+        // Append the headers of the CSV content.
+        builder.append("uuid,bundle,application,eventType,created");
+        builder.append(System.lineSeparator());
+
+        for (final Event event : events) {
+            builder.append(
+                String.format(
+                    "%s,%s,%s,%s,%s%s",
+                    event.getId(),
+                    event.getBundleDisplayName(),
+                    event.getApplicationDisplayName(),
+                    event.getEventTypeDisplayName(),
```

Comment:
What if a comma is used in a display name? Commas are supported in CSV values as long as these values are wrapped into double quotes. We should either support that or forbid commas in display names.

Reply:
We should search for csv library that support these features, it can get tricky if we want to support other data (such as descriptions) that can already have commas and double quotes.

Reply:
Yeah, good idea. I thought we could do it without having to deal with exceptions here but of course there are many cases I didn't think about... So I've used Apache's CSV library, which implements [RFC 4180](https://datatracker.ietf.org/doc/html/rfc4180) for CSVs, among others.